src/custom/state/claim/hooks/index.ts

```diff
@@ -62,7 +62,7 @@ import useIsMounted from 'hooks/useIsMounted'
 import { ChainId } from '@uniswap/sdk'
 import { ClaimInfo } from 'state/claim/reducer'
 
-const CLAIMS_REPO_BRANCH = 'gip-13'
+const CLAIMS_REPO_BRANCH = '2022-02-15_Rinkeby-full'
```

Comment:
Note to self: revert this commit before merging, we don't want to change the claims repo branch